generators/beamAndVertexGenerator.cc

```diff
@@ -1,13 +1,18 @@
 
 #include <TFile.h>
 #include <TTree.h>
+#include <TVectorD.h>
+#include <TMatrixDSymEigen.h>
 
 #include "generator.h"
 #include "generatorParameters.hpp"
 #include "randomNumberGenerator.h"
 #include "reportingUtils.hpp"
 #include "beamAndVertexGenerator.h"
 
+namespace {
+	TVectorD multivariateGauss(const TVectorD& mean, const TMatrixDSym& cov, const double sigmaScalingFactor);
```

Comment:
Probably useful also in other places. Would it make sense to move this to mathUtils.hpp?
Set default sigmaScalingFactor = 1?